includes/admin/settings.php

```
@@ -209,11 +241,102 @@ function settings_page(): void {
+		<div class="knabbel-articles-footer">
+			<form method="post" action="<?php echo esc_url( admin_url( 'admin-post.php' ) ); ?>">
+				<input type="hidden" name="action" value="knabbel_clear_recent_errors" />
+				<?php wp_nonce_field( 'knabbel_clear_recent_errors' ); ?>
+				<button type="submit" class="knabbel-btn knabbel-btn-secondary">
+					<?php esc_html_e( 'Clear Errors', 'zw-knabbel-wp' ); ?>
+				</button>
+			</form>
+		</div>
+	</div>
+	<?php
+}
```

PHPMD (2.15.0)

[warning] The function render_recent_errors() has a Cyclomatic Complexity of 15. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)